UI/window-basic-main.cpp

```diff
@@ -7427,6 +7427,13 @@ void OBSBasic::on_actionDiscord_triggered()
 	QDesktopServices::openUrl(url);
 }
 
+void OBSBasic::on_actionReleaseNotes_triggered()
+{
+	QString addr("https://github.com/obsproject/obs-studio/releases");
+	QUrl url(QString("%1/%2").arg(addr, OBS_VERSION), QUrl::TolerantMode);
```

Comment:
With #8930 merged this should use `obs_get_version_string()` instead. Or alternatively `OBSApp::GetVersionString()´ which will also work if `OBS_VERSION` is not defined.

Reply:
Ended up using `obs_get_version_string`, seems to work fine.